### ISSUE TITLE
Update README with more helpful install install instructions

### ISSUE THREAD

**Issue by benthayer** [benthayer/git-gud#199]
In general, follow something along these steps:

1. Install Git
2. Install Anaconda
3. Open Anaconda prompt (on windows)
4. pip install git-gud
5. mkdir test
6. cd test
7. git gud init

Make sure instructions work on windows mac and linux

**Comment by sahansk2** [comment]
@benthayer Scoop

**Comment by benthayer** [comment]
You used Scoop twice

**Comment by sahansk2** [comment]
What can I say, I'm a two-scoops kinda guy. :^)

**Comment by sahansk2** [comment]
Note: I can only write a guide for Windows, since I do not have a macOS workstation. Contributions to provide a macOS installation guide are welcome.

We should also recommend the use of Miniconda over Anaconda. Anaconda is much larger and not necessary for Git Gud.

**Comment by benthayer** [comment]
I agree. We can recommend miniconda and say (a full anaconda install is
will work too)

**Comment by sahansk2** [comment]
Relevant, so I'll put this here: I put together a non-WSL solution to installing Git Gud on Windows, feel free to use and adapt: https://open-source-at-illinois.github.io/workshops/git-gg
